text-utilities.h

```diff
@@ -36,6 +36,14 @@ static void set_font(struct pango_source *src, PangoLayout *layout) {
 			FcFontSet *font_set = FcFontSetCreate();
 			int count = FcFreeTypeQueryAll(src->font_file, -1, NULL, &count, font_set);
 			if (count > 0 ) {
+#if defined(PANGO_VERSION_CHECK) && PANGO_VERSION_CHECK(1,38,0)
+				// need to explicitly notify newer Pango versions that a new font has been added to the FC map
+				// more info: https://gitlab.gnome.org/GNOME/gtk/-/issues/3886
+				if (PANGO_IS_FC_FONT_MAP(pango_cairo_font_map_get_default())) {
+					blog(LOG_INFO, "[pango] refreshing fontconfig map");
```

Comment:
nit: I dont think should be logged, reloading font caches is performant enough these days.

Reply:
Removed the logging statement in 5f13c95.